bi-connectors/PowerBIConnector/README.md

```diff
@@ -0,0 +1,23 @@
+## Connector Download
+
+The most recent build could be donwloaded from an automated CI workflow: [link](https://github.com/opensearch-project/sql/actions/workflows/bi-connectors.yml).

```

Comment:
`The most recent build could be donwloaded from an automated CI workflow`
to:
`The Power BI connector is available to be downloaded from the automated CI workflow`